tests/http_client_test.cc

```diff
@@ -86,12 +88,15 @@ TEST(http_client_test, one_client_with_multiple_requests) {
     int response_counter = 0;
 
     auto rb = client.get(server_address);
-    for (int i = 0; i < RESPONSE_SIZE; ++i) {
-        auto response = rb.header<Http::Header::Connection>(Http::ConnectionControl::KeepAlive).send();
-        response.then([&](Http::Response rsp) {
-            if (rsp.code() == Http::Code::Ok)
-                ++response_counter;
-        }, Async::IgnoreException);
```

Comment:
Please try to avoid reformatting this code when making logic changes, it adds to the cognitive burden of trying to decipher what has changed and the broader implications of the change.

Reply:
@dennisjenkins75 I see, I'm sorry, should I revert change in this commit related to this comment?

Reply:
You do not need to change this PR.  But please avoid reformatting code not directly related to the logic changes that you are making.

Does your editor do this automatically?

It would be a problem if different contributor's editors constantly fought over style.  Arthur asked me about adopting clang-tidy as an enforced formatting style.  I don't want to do that just yet; forceably reformatting all of the code will break the "blame" for each line, and break everyone's inflight PRs.  The community as a whole should decide to do this.  Discussion for another day.

We really need a useable web forum and not a disjoint IRC and Slack channels.

Reply:
@knowledge4igor I caught your point. I hope one day I will install IRC or Slack and join to discussion, maybe at the weekend. Is it a good idea to discuss formatting style in issues here (as we were discussing C++14 and C++17) ?

Reply:
@dennisjenkins75 I caught your point. I hope one day I will install IRC or Slack and join to discussion, maybe at the weekend. Is it a good idea to discuss formatting style in issues here (as we were discussing C++14 and C++17) ?